x-pack/plugins/maps/public/embeddable/map_embeddable_factory.js

```diff
@@ -9,11 +9,21 @@ import chrome from 'ui/chrome';
 import { EmbeddableFactory } from 'ui/embeddable';
 import { MapEmbeddable } from './map_embeddable';
 import { indexPatternService } from '../kibana_services';
+import { i18n } from '@kbn/i18n';
 
 export class MapEmbeddableFactory extends EmbeddableFactory {
 
   constructor(gisMapSavedObjectLoader) {
-    super({ name: 'map' });
+    super({
+      name: 'map',
+      savedObjectMetaData: {
+        name: i18n.translate('xpack.maps.mapSavedObjectLabel', {
+          defaultMessage: 'Map',
+        }),
+        type: 'map',
```

Comment:
This type is getting a little scattered. Would you mind extracting `map` to a constant and use it here as well?

- https://github.com/thomasneirynck/kibana/blob/55e7b183d7530c3ea279338a73e2e800b228db6c/x-pack/plugins/maps/server/maps_telemetry/maps_telemetry.js#L93
- https://github.com/thomasneirynck/kibana/blob/54bbdbaedbb4d17b1479630b2e308fd6dbbcd388/x-pack/plugins/maps/public/angular/services/saved_gis_map.js#L66
- https://github.com/thomasneirynck/kibana/blob/cbe114856adcf58321609c35e2edc0df6b3b1510/x-pack/plugins/maps/public/angular/map_controller.js#L313